lib/scarpe/wv/star.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class Scarpe
+  class WebviewStar < Scarpe::WebviewWidget
+    def initialize(properties)
+      super(properties)
+    end
+
+    def element(&block)
+      HTML.render do |h|
+        h.div(id: html_id, style: style) do
+          h.svg(width: @outer, height: @outer, style: "fill:#{fill_color};") do
```

Comment:
Oh, interesting! I'm used to using SVGs as big chunks like graphs or dense rectangles like icons. I haven't tried layering them with other stuff, but I assume it works. Neat!